soccer/Robot.hpp

```diff
@@ -430,19 +470,23 @@ class OurRobot: public Robot {
 	RobotMask _self_avoid_mask, _opp_avoid_mask;  /// masks for obstacle avoidance
 	float _avoidBallRadius; /// radius of ball obstacle
 
+	Planning::MotionCommand _motionCommand;
+	Planning::MotionCommand::CommandType _lastCommandType;
 	MotionConstraints _motionConstraints;
 
-	Planning::RRTPlanner *_planner;	/// single-robot RRT planner
+	std::shared_ptr<Planning::PathPlanner> _planner;	/// single-robot RRT planner
```

Comment:
Alright, I guess so.  In general it's best to write things for the way they are now rather than speculating on how we might use them in the future though :)